tests/optim/test_ddp_adascale.py

```diff
@@ -7,7 +7,19 @@
 # pylint: disable=missing-class-docstring
 # pylint: disable=missing-function-docstring
 
-""" Test AdaScale with DDP. """
+""" Test AdaScale with DDP/SDP/FSDP.
+
+    Even though it is tested here, AdaScale does NOT work with SDP/FSDP the
```

Comment:
this is the main note.

mainly this change is to add slight testing so that we know the status of things. I feel that no need to spend nontrivial amount of effort until a real use case shows up for these combinations.